internal/controller/assets/daemonset.go

```diff
@@ -213,7 +213,7 @@ func Daemonset(dsName, image, serviceAccount string, node *falconv1alpha1.Falcon
 					// NodeSelector is set to linux until windows containers are supported for the Falcon sensor
 					NodeSelector:                  common.NodeSelector,
 					Affinity:                      nodeAffinity(node),
-					Tolerations:                   node.Spec.Node.Tolerations,
+					Tolerations:                   *node.Spec.Node.Tolerations,
```

Comment:
Would recommend you make this a function that returns an empty toleration if nil

Reply:
@redhatrises Updated to a function to check for nil.